File: 1346. Check If N and Its Double Exist.go
func checkIfExist(arr []int) bool {
     n := len(arr)
    for i := 0; i < n; i++ {
        for j := i + 1; j < n; j++ {
            if arr[i]*2 == arr[j] || arr[j]*2 == arr[i] {return true}
        }
    }
    return false
}

File: 1346. Check If N and Its Double Exist.kt
class Solution {
    fun checkIfExist(arr: IntArray): Boolean {
        val n = arr.size
        for (i in 0 until n) {
            for (j in i + 1 until n) {
                if (arr[i] * 2 == arr[j] || arr[j] * 2 == arr[i]) { return true }
            }
        }
        return false
    }
}
